### PR TITLE
1040 Topic's header ReBrand changes

### LINKED ISSUE #1040: [Rebrand] Header on the Topic pages
https://www.figma.com/file/fgBV6T1rkYBvzQYzwwUnFt/Bioscience-hub-3?type=design&node-id=25345-356339&mode=design&t=ljJC8BG0mgag3eVD-4

image

### PR DESCRIPTION
**Pull request by amlan18041996**
Fix #1040 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/us/en/products/mass-spectrometers/topics/what-is-mass-spectrometry
- After: https://1040-header-topic-pages--danaher-ls-aem--hlxsites.hlx.page/us/en/products/mass-spectrometers/topics/what-is-mass-spectrometry
- After: https://1040-header-topic-pages--danaher-ls-aem--hlxsites.hlx.page/us/en/products/flow-cytometers/topics/sheath-fluid-flow-cytometry

### CODE REVIEW

**Review by rgravitvl — APPROVED** [review]

**Review by sharanagoudapatil — APPROVED** [review]
Approved. But one small suggestion - can we make use of the right side space ?  
Please check this page
image

**Comment by amlan18041996** [comment]
Changes applied 50% based on figma design

File: blocks/side-nav/side-nav.js [hunk 1]
```diff
@@ -31,18 +31,19 @@ function renderSideNav(sideNavItems) {
 }
 
 export default async function decorate(block) {
+  console.log(block, block.parentElement);
```
Comment (rgravitvl) [inline blocks/side-nav/side-nav.js:34]: Remove console log

**Review by rgravitvl — CHANGES_REQUESTED** [review]

**Comment by davenichols-DHLS** [comment]
I am approving, but the CLS appears worse on this change than on current prod.

Current:  https://pagespeed.web.dev/analysis/https-lifesciences-danaher-com-us-en-products-flow-cytometers-topics-sheath-fluid-flow-cytometry-html/co467ygogc?form_factor=mobile
This Branch:  https://pagespeed.web.dev/analysis/https-1040-header-topic-pages--danaher-ls-aem--hlxsites-hlx-page-us-en-products-flow-cytometers-topics-sheath-fluid-flow-cytometry/dpl8it3ede?form_factor=mobile

**Review by davenichols-DHLS — APPROVED** [review]
Approved.  @amlan18041996  - can you please fix the conflicts.

**Comment by davenichols-DHLS** [comment]
@amlan18041996  - are you able to please fix the conflicts and then I will merge this?  Thanks.

**Pull request merged**